source/backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -97,33 +88,44 @@ private static ExpressionStarter<PimsProperty> GenerateCommonPropertyQuery(Claim
 
             var isRetired = filter.Ownership.Contains("isRetired");
 
-            ExpressionStarter<PimsProperty> ownershipBuilder;
+            ExpressionStarter<PimsPropertyLocationVw> ownershipBuilder;
 
             if (filter.Ownership.Count > 0)
             {
-                ownershipBuilder = isRetired ? PredicateBuilder.New<PimsProperty>(p => p.IsRetired == true) : PredicateBuilder.New<PimsProperty>(p => false);
+                var authorizationTypes = new List<string>() {
```

Comment:
nit, this isn't all the types, just the types that result in interests, could update naming to better reflect that.

Also, do we not have an enum for this?

Reply:
Is this being used?

Reply:
Same as below. I switched the implementation of the filters halfway to use the view instead of replicating the query